plugin/completion/lib_complete.py

```diff
@@ -388,7 +396,7 @@ def _parse_completions(complete_results, excluded):
                     continue
                 hint += chunk.spelling
                 if chunk.isKindTypedText():
-                    trigger = chunk.spelling
+                    trigger += chunk.spelling
```

Comment:
Yeah, this is intended, see message at https://github.com/niosus/EasyClangComplete/pull/219/commits/c9325c59ad05a2c84f85a2fff7f5fabf4cdf9fcf
As far as I could tell, C/C++ only hits this line once, so it doesn't matter if we use = or +=.